# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Bujar Tafili, 2015-2016
# Genti Ereqi, 2014
# THick Atmosphere <co404@protonmail.ch>, 2018
msgid ""
msgstr ""
"Project-Id-Version: The Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-08-16 11:40+0200\n"
"PO-Revision-Date: 2018-08-16 14:46+0000\n"
"Last-Translator: carolyn <carolyn@anhalt.org>\n"
"Language-Team: Albanian (http://www.transifex.com/otf/torproject/language/sq/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: sq\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../tails_installer/creator.py:100
msgid "You must run this application as root"
msgstr "Duhet ta ekzekutoni këtë program si rrënjë"

#: ../tails_installer/creator.py:146
msgid "Extracting live image to the target device..."
msgstr "Nxjerrja e imazhit të drejtpërdrejtë në pajisjen synuar..."

#: ../tails_installer/creator.py:153
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "Shkroi në pajisje me shpejtësi %(speed)d MB/sek"

#: ../tails_installer/creator.py:296
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "U has një problem gjatë ekzekutimit të komandës vijuese: `%(command)s`.\nNjë regjistër gabimi më i hollësishëm është shkruar tek '%(filename)s'."

#: ../tails_installer/creator.py:315
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "Duke verifikuar shumën e saktë të shifrave SHA1 të imazhit të LiveCD"

#: ../tails_installer/creator.py:319
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "Duke verifikuar shumën e saktë të shifrave SHA256 të imazhit të LiveCD"

#: ../tails_installer/creator.py:335
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "Gabim: SHA1 e Live CD-së tuaj është e pavlefshme. Ju mund ta ekzekutoni këtë program me --argumentin \"mos verifikoni\", për të shmangur kontrollin e verifikimit."

#: ../tails_installer/creator.py:341
msgid "Unknown ISO, skipping checksum verification"
msgstr "ISO e panjohur, duke kapërcyer verifikimin e shumës së saktë të shifrave"

#: ../tails_installer/creator.py:353
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr "S'ka hapësirë të mjaftueshme në pajisje.\n%dMB ISO + mbivendosja %dMB > hapësira e lirë %dMB"

#: ../tails_installer/creator.py:360
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "Duke krijuar mbulim të vazhdueshëm të %sMB "

#: ../tails_installer/creator.py:421
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "S'mund të kopjohet %(infile)s tek %(outfile)s: %(message)s"

#: ../tails_installer/creator.py:435
msgid "Removing existing Live OS"
msgstr "Largimi i Live OS-së ekzistuese"

#: ../tails_installer/creator.py:444 ../tails_installer/creator.py:456
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "S'mund të përdoret regjimi i ndryshimit të komandës %(file)s: %(message)s"

#: ../tails_installer/creator.py:449
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "S'mund të hiqet skedari nga LiveOS i mëparshëm: %(message)s"

#: ../tails_installer/creator.py:462
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "S'mund të hiqet direktoria nga LiveOS i mëparshëm: %(message)s"

#: ../tails_installer/creator.py:510
#, python-format
msgid "Cannot find device %s"
msgstr "S'mund të gjendet pajisja %s"

#: ../tails_installer/creator.py:711
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "S'mund të shkruhet tek %(device)s, duke e kapërcyer."

#: ../tails_installer/creator.py:741
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr "Disa njësi të ndara të pajisjes së synuar %(device)s janë montuar. Ato do të çmontohen përpara se të nisë procesi i instalimit."

#: ../tails_installer/creator.py:784 ../tails_installer/creator.py:1008
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "Sistem i panjohur skedarësh. Pajisja juaj mund të ketë nevojë për riformatim."

#: ../tails_installer/creator.py:787 ../tails_installer/creator.py:1011
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Sistem skedarësh i pa mbështetur: %s"

#: ../tails_installer/creator.py:805
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr ""

#: ../tails_installer/creator.py:810
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "S'mund të montohet pajisja: %(message)s"

#: ../tails_installer/creator.py:815
msgid "No mount points found"
msgstr "S'u gjet asnjë pikë montimi"

#: ../tails_installer/creator.py:826
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr "Duke futur çmontoni_pajisjen për '%(device)s'"

#: ../tails_installer/creator.py:836
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr "Çmontimi i sistemit të skedarëve të montuar në '%(device)s'"

#: ../tails_installer/creator.py:840
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr "Çmontimi i '%(udi)s' në '%(device)s'"

#: ../tails_installer/creator.py:851
#, python-format
msgid "Mount %s exists after unmounting"
msgstr "Montimi i %s është i pranishëm pas çmontimit"

#: ../tails_installer/creator.py:864
#, python-format
msgid "Partitioning device %(device)s"
msgstr "Pajisja ndarëse %(device)s"

#: ../tails_installer/creator.py:993
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr "Pajisje '%(device)s' e pamontuar. Ju lutemi raportoni një gabim."

#: ../tails_installer/creator.py:996
msgid "Trying to continue anyway."
msgstr "Duke u rrekur për të vazhduar gjithsesi."

#: ../tails_installer/creator.py:1005 ../tails_installer/creator.py:1401
msgid "Verifying filesystem..."
msgstr "Duke verifikuar sistemin e skedarëve..."

#: ../tails_installer/creator.py:1029
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "S'mund të ndryshohet etiketa e volumit: %(message)s"

#: ../tails_installer/creator.py:1034 ../tails_installer/creator.py:1434
msgid "Installing bootloader..."
msgstr "Duke instaluar nisësin..."

#: ../tails_installer/creator.py:1061
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr ""

#: ../tails_installer/creator.py:1069 ../tails_installer/creator.py:1452
#, python-format
msgid "Removing %(file)s"
msgstr "Largimi i %(file)s"

#: ../tails_installer/creator.py:1183
#, python-format
msgid "%s already bootable"
msgstr "%s plotësisht funksional"

#: ../tails_installer/creator.py:1203
msgid "Unable to find partition"
msgstr "S'mund të gjendet njësia e ndarjë"

#: ../tails_installer/creator.py:1226
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "Duke e formatuar %(device)s si FAT32"

#: ../tails_installer/creator.py:1286
msgid "Could not find syslinux' gptmbr.bin"
msgstr "Nuk u gjet syslinux' gptmbr.bin"

#: ../tails_installer/creator.py:1299
#, python-format
msgid "Reading extracted MBR from %s"
msgstr ""

#: ../tails_installer/creator.py:1303
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr ""

#: ../tails_installer/creator.py:1316 ../tails_installer/creator.py:1317
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "Rivendosja e Regjistrit të Nisjes Kryesore për %s"

#: ../tails_installer/creator.py:1322
msgid "Drive is a loopback, skipping MBR reset"
msgstr "Pajisja është riciklike, duke shmangur rivendosjen e MBR"

#: ../tails_installer/creator.py:1326 ../tails_installer/creator.py:1580
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "Duke përllogaritur SHA1 të %s"

#: ../tails_installer/creator.py:1351
msgid "Synchronizing data on disk..."
msgstr "Duke sinkronizuar të dhënat në disk..."

#: ../tails_installer/creator.py:1393
msgid "Error probing device"
msgstr "Pajisje për sondimin e gabimit"

#: ../tails_installer/creator.py:1395
msgid "Unable to find any supported device"
msgstr "S'mund të gjendet asnjë pajisje e pranuar"

#: ../tails_installer/creator.py:1405
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Sigurohuni që USB-ja juaj të jetë e ngulur dhe e formatuar me sistemin e skedarëve FAT"

#: ../tails_installer/creator.py:1408
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "Sistem skedarësh i papranueshëm: %s\nJu lutemi ruajeni dhe formatoni diskun tuaj USB, me sistemin e skedarëve FAT."

#: ../tails_installer/creator.py:1475
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr "S'mund të merret Win32_LogicalDisk; pyetje win32com nuk ktheu asnjë rezultat"

#: ../tails_installer/creator.py:1528
msgid "Cannot find"
msgstr "S'mund të gjendet"

#: ../tails_installer/creator.py:1529
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr "Sigurohuni ta ekstraktoni të gjithë tails-installer skedarin zip para se ta ekzekutoni këtë program."

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Version i panjohur: %s"

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "Duke shkarkuar %s..."

#: ../tails_installer/gui.py:213
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "Gabim: S'mund të vendoset etiketa ose të përftohet UUID-i i pajisjes tuaj. I pamundur vazhdimi."

#: ../tails_installer/gui.py:260
#, python-format
msgid "Installation complete! (%s)"
msgstr "Instalimi u krye! (%s)"

#: ../tails_installer/gui.py:265
msgid "Tails installation failed!"
msgstr ""

#: ../tails_installer/gui.py:361
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "Kujdes: Ky mjet lypset të ekzekutohet si një Administrator. Për ta bërë këtë, klikoni me të djathtën mbi ikonë dhe hapni Vetitë. Tek butoni Përputhshmëria, shenjoni kutizën \"Ekzekutojeni këtë program si administrator\"."

#: ../tails_installer/gui.py:373
msgid "Tails Installer"
msgstr ""

#: ../tails_installer/gui.py:423 ../data/tails-installer.ui.h:2
msgid "Clone the current Tails"
msgstr ""

#: ../tails_installer/gui.py:430 ../data/tails-installer.ui.h:3
msgid "Use a downloaded Tails ISO image"
msgstr ""

#: ../tails_installer/gui.py:467
msgid "Upgrade"
msgstr ""

#: ../tails_installer/gui.py:469
msgid "Manual Upgrade Instructions"
msgstr ""

#: ../tails_installer/gui.py:471
msgid "https://tails.boum.org/upgrade/"
msgstr ""

#: ../tails_installer/gui.py:479 ../data/tails-installer.ui.h:7
msgid "Install"
msgstr ""

#: ../tails_installer/gui.py:482 ../data/tails-installer.ui.h:1
msgid "Installation Instructions"
msgstr ""

#: ../tails_installer/gui.py:484
msgid "https://tails.boum.org/install/"
msgstr ""

#: ../tails_installer/gui.py:490
#, python-format
msgid "%(size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:502
msgid "No ISO image selected"
msgstr ""

#: ../tails_installer/gui.py:503
msgid "Please select a Tails ISO image."
msgstr ""

#: ../tails_installer/gui.py:545
msgid "No device suitable to install Tails could be found"
msgstr ""

#: ../tails_installer/gui.py:547
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr ""

#: ../tails_installer/gui.py:581
#, python-format
msgid ""
"The USB stick \"%(pretty_name)s\" is configured as non-removable by its "
"manufacturer and Tails will fail to start on it. Please try installing on a "
"different model."
msgstr ""

#: ../tails_installer/gui.py:591
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr "Pajisja \"%(pretty_name)s\" është shumë e vogël për të instaluar Tails (duhen të paktën %(size)s GB)."

#: ../tails_installer/gui.py:604
#, python-format
msgid ""
"To upgrade device \"%(pretty_name)s\" from this Tails, you need to use a downloaded Tails ISO image:\n"
"https://tails.boum.org/install/download"
msgstr ""

#: ../tails_installer/gui.py:625
msgid "An error happened while installing Tails"
msgstr ""

#: ../tails_installer/gui.py:637
msgid "Refreshing releases..."
msgstr "Duke rifreskuar versionet..."

#: ../tails_installer/gui.py:642
msgid "Releases updated!"
msgstr "Versionet u përditësuan!"

#: ../tails_installer/gui.py:695
msgid "Installation complete!"
msgstr "Instalimi u krye!"

#: ../tails_installer/gui.py:744
msgid "Unable to mount device"
msgstr "S'mund të montohet pajisja"

#: ../tails_installer/gui.py:751 ../tails_installer/gui.py:783
msgid "Confirm the target USB stick"
msgstr ""

#: ../tails_installer/gui.py:752
#, python-format
msgid ""
"%(size)s %(vendor)s %(model)s device (%(device)s)\n"
"\n"
"All data on this USB stick will be lost."
msgstr ""

#: ../tails_installer/gui.py:770
#, python-format
msgid "%(parent_size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:778
msgid ""
"\n"
"\n"
"The persistent storage on this USB stick will be preserved."
msgstr ""

#: ../tails_installer/gui.py:779
#, python-format
msgid "%(description)s%(persistence_message)s"
msgstr ""

#: ../tails_installer/gui.py:821
msgid "Download complete!"
msgstr "Shkarkimi u krye!"

#: ../tails_installer/gui.py:825
msgid "Download failed: "
msgstr "Shkarkimi dështoi:"

#: ../tails_installer/gui.py:826
msgid "You can try again to resume your download"
msgstr "Mund të provoni sërish që ta rinisni shkarkimin tuaj."

#: ../tails_installer/gui.py:834
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "Skedari i përzgjedhur është i palexueshëm. Ju lutemi ndreqini lejet e tij, ose përzgjidhni një skedar tjetër."

#: ../tails_installer/gui.py:840
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "S'mund të përdoret skedari i përzgjedhur. Mund të jeni më me fat nëse e lëvizni ISO-n tuaj, tek rrënja e diskut tuaj (psh: C:\\)"

#: ../tails_installer/gui.py:846
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s e përzgjedhur"

#: ../tails_installer/source.py:28
msgid "Unable to find LiveOS on ISO"
msgstr ""

#: ../tails_installer/source.py:34
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr ""

#: ../tails_installer/source.py:49
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr ""

#: ../tails_installer/source.py:63
#, python-format
msgid "'%s' does not exist"
msgstr ""

#: ../tails_installer/source.py:65
#, python-format
msgid "'%s' is not a directory"
msgstr ""

#: ../tails_installer/source.py:75
#, python-format
msgid "Skipping '%(filename)s'"
msgstr ""

#: ../tails_installer/utils.py:44
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr ""

#: ../tails_installer/utils.py:124
msgid "Could not open device for writing."
msgstr ""

#: ../data/tails-installer.ui.h:4
msgid "Select a distribution to download:"
msgstr ""

#: ../data/tails-installer.ui.h:5
msgid "Target USB stick:"
msgstr ""

#: ../data/tails-installer.ui.h:6
msgid "Reinstall (delete all data)"
msgstr ""
